calc.js

```diff
@@ -1,3 +1,124 @@
 // Build a simple webpage that has an interactive calculator using JavaScript
 
-// You will want some kind of html element to display the calculations (maybe a span) and a few buttons to press to input numbers and operators
\ No newline at end of file
+// You will want some kind of html element to display the calculations (maybe a span) and a few buttons to press to input numbers and operators
+
+let input = document.getElementById("Input");
+let history = "";
+const divide = document.getElementById("Divide");
+const multiply = document.getElementById("Multiply");
+const subtract = document.getElementById("Subtract");
+const addition = document.getElementById("Addition");
+const seven = document.getElementById("Seven");
+const eight = document.getElementById("Eight");
+const nine = document.getElementById("Nine");
+const clear = document.getElementById("Clear");
+const four = document.getElementById("Four");
+const five = document.getElementById("Five");
+const six = document.getElementById("Six");
+const allClear = document.getElementById("allClear");
+const one = document.getElementById("One");
+const two = document.getElementById("Two");
+const three = document.getElementById("Three");
+const equal = document.getElementById("Equal");
+const zero = document.getElementById("Zero");
+
+input.addEventListener("click", useValue);
```

Comment:
if you use the above method you could for a for loop and add an event listener to each element in the array, instead of writing it out a million times